Demo/Application/Features/Preferred Payment Methods/BraintreeDemoPreferredPaymentMethodsViewController.swift

```diff
@@ -0,0 +1,132 @@
+        venmoButton.translatesAutoresizingMaskIntoConstraints = false
+        venmoButton.addTarget(self, action: #selector(venmoButtonTapped(_:)), for: .touchUpInside)
+        venmoButton.isEnabled = false
+        view.addSubview(venmoButton)
+        
+        view.addConstraints([preferredPaymentMethodsButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+                             preferredPaymentMethodsButton.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+                             payPalVaultButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+                             payPalVaultButton.bottomAnchor.constraint(equalTo: payPalCheckoutButton.bottomAnchor, constant: -40),
+                             payPalCheckoutButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+                             payPalCheckoutButton.bottomAnchor.constraint(equalTo: venmoButton.bottomAnchor, constant: -40),
+                             venmoButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+                             venmoButton.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -40)])
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    @objc func preferredPaymentMethodsButtonTapped(_ button: UIButton) {
+        self.progressBlock("Fetching preferred payment methods...")
+        preferredPaymentMethods.fetch { (result) in
+            self.progressBlock("PayPal Preferred: \(result.isPayPalPreferred)\nVenmo Preferred: \(result.isVenmoPreferred)")
+            self.payPalCheckoutButton.isEnabled = result.isPayPalPreferred
+            self.payPalVaultButton.isEnabled = result.isPayPalPreferred
+            self.venmoButton.isEnabled = result.isVenmoPreferred
+        }
+    }
+    
+    @objc func payPalCheckoutButtonTapped(_ button: UIButton) {
+        self.progressBlock("Tapped PayPal Checkout")
+        
+        button.setTitle("Processing...", for: .disabled)
+        button.isEnabled = false
+        
+        let paypalRequest = BTPayPalCheckoutRequest(amount: "4.30")
+        paypalClient.tokenizePayPalAccount(with: paypalRequest) { (nonce, error) in
+            button.isEnabled = true
+
+            if let e = error {
+                self.progressBlock(e.localizedDescription)
+            } else if let n = nonce {
+                self.nonceStringCompletionBlock(n.nonce)
+            } else {
+                self.progressBlock("Canceled")
+            }
+        }
+    }
+    
+    @objc func payPalVaultButtonTapped(_ button: UIButton) {
+        self.progressBlock("Tapped PayPal Vault")
+        
+        button.setTitle("Processing...", for: .disabled)
+        button.isEnabled = false
+        
+        let paypalRequest = BTPayPalVaultRequest()
+        paypalRequest.activeWindow = self.view.window
+        paypalClient.tokenizePayPalAccount(with: paypalRequest) { (nonce, error) in
```

Comment:
Actually are these methods correct? They should all be `tokenize(request)`? This all looks like the old v5 syntax.

Reply:
Seems like maybe this entire ViewController could be old and committed on accident?

Reply:
Yeh this VC is real old